CMakeLists.txt

```diff
@@ -77,6 +81,19 @@ endif()
 #
 # Optimizations
 #
+
+# CMAKE_INTERPROCEDURAL_OPTIMIZATION can be defined to override the default behaviour.
+# We keep DEFINED CMAKE_INTERPROCEDURAL_OPTIMIZATION unset = FALSE for Debug builds to save
+# build time during development at the cost of a bigger memory foodprint.
```

Comment:
Is the higher memory requirement at build time or run time? Please clarify the comment.

Reply:
At run time.